imports/ui/accounts/Account.jsx

```diff
@@ -683,8 +683,8 @@ export default class AccountDetails extends Component {
                                     {this.state.user ? <Row>
                                         <Col xs={12} >
                                             {this.state.user === this.state.address ? <ClaimSwapButton validator={this.props.validator} address={this.state.operator_address} history={this.props.history} /> : null}
-                                            {this.state.user === this.state.address ? <WithdrawButton history={this.props.history} rewards={this.state.rewards} commission={this.state.commission} address={this.state.operator_address} denom={this.state.denom} /> : null}
-                                            <TransferButton history={this.props.history} address={this.state.address} denom={this.state.denom} total={this.state.total} /></Col>
+                                            {this.state.user === this.state.address ? <WithdrawButton rewards={this.state.rewards} commission={this.state.commission} address={this.state.operator_address} denom={this.state.denom} /> : null}
```

Comment:
Why `WithdrawButton` and `TransferButton` don't need the `history` prop but `ClaimSwapButton` needs it? What is the original usage of the `history` prop?

Reply:
Oops, I have updated it now. The history prop is used in LedgerButton component, hence I assume the previous developer working on it passed history prop by mistake or needed it in the past, but forgot to clean up the code after.

Reply:
Hm... if it's being used in the `LedgerButton` then it should be in all these 3 buttons as they are inherited from `LedgerButton`.